impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -110,6 +113,24 @@ ISize AllocatorVK::GetMaxTextureSizeSupported() const {
   return max_texture_size_;
 }
 
+void AllocatorVK::CheckForMemoryTypeSupport() {
```

Comment:
IMO, this should just be a cap check set in `CapabilitiesVK::SetPhysicalDevice`. That way, if we decide this is a dealbreaker for Vulkan, we can decide to more easily terminate context setup and fallback to OpenGL. The check here is too late to make that call.